sentinel/db-pouch.js

```diff
@@ -0,0 +1,11 @@
+const PouchDB = require('pouchdb-core');
+PouchDB.plugin(require('pouchdb-adapter-http'));
+PouchDB.plugin(require('pouchdb-mapreduce'));
+
+const { COUCH_URL } = process.env;
+// strip trailing slash from to prevent bugs in path matching
+const couchUrl = COUCH_URL && COUCH_URL.replace(/\/$/, '');
+
+module.exports.medic = new PouchDB(couchUrl);
+module.exports.audit = new PouchDB(`${couchUrl}-audit`);
+module.exports._url = couchUrl;
```

Comment:
Same comment as the api version. Maybe we need a db shared-lib? ;)

Reply:
Yeah I was going to pull these out afterwards. One thing at a time.